Cargo.toml

```diff
@@ -34,6 +34,7 @@ members = [
     "libs/error-stack",
     "libs/error-stack/macros",
     "libs/sarif",
+    "libs/chonky",
```

Comment:
@TimDiekmann Can we alphabetize?

Reply:
Sure! [`2d20beb` (#5475)](https://github.com/hashintel/hash/pull/5475/commits/2d20bebe6bac936a0a3aa0b01036f0136a46c36a)